packages/node-bridge/bridge.js

```diff
@@ -1,4 +1,10 @@
+const { URL } = require('url');
```

Comment:
I think you can use the global `URL` instead

Reply:
TypeScript doesn't seem to like it, so seems like something isn't configured correctly.

Reply:
I also ran into not being able to use the global `URL`. Maybe our `@types/node` is out of date?